syntax = "proto3";

package csm.service.protocol;

message HeartBeat
{
    string nodeId = 1;
    uint64 seq = 2;
    uint32 timestamp = 3;
}

message HeartBeatReply
{
    string nodeId = 1;
    uint64 seq = 2;
    uint32 transferTime = 3;
}

